libcloud/compute/types.py

```diff
@@ -207,25 +207,19 @@ class NodeState(object):
     :cvar PAUSED: Node is paused.
     :cvar UNKNOWN: Node state is unknown.
     """
-    RUNNING = 0
-    REBOOTING = 1
-    TERMINATED = 2
-    PENDING = 3
-    UNKNOWN = 4
-    STOPPED = 5
-    SUSPENDED = 6
-    ERROR = 7
-    PAUSED = 8
+    RUNNING = "running"
+    REBOOTING = "rebooting"
```

Comment:
Minor style thing - for consistency, please use single quotes around simple strings :)